internal/binary.bzl

```diff
@@ -50,11 +50,10 @@ def postcss_binary(
             * `bazel.additionalOutputs`: An array of all the file paths passed
               to the `additional_outputs` attribute of `postcss_binary`, to
               which a plugin is expected to write.
-
-        deps: A list of NodeJS modules the config depends on. The PostCSS module
-            is always implicitly included.
         src: The input .css, and optionally .css.map files. (This includes
             outputs from preprocessors such as sass_binary.)
+        deps: A list of NodeJS modules the config depends on. The PostCSS module
+            is always implicitly included.
```

Comment:
It's probably worth mentioning that all plugin modules are implicitly included.

Reply:
Done.